### ISSUE TITLE
Erasing grouped objects causes tdraw to display an error (2.6-beta-5)

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#16428]
**Describe the bug**
In 2.6.0-beta5, grouping two objects together and using the eraser causes a crash.

**To Reproduce**
1. Create two objects
2. Group them together
3. Erase them
4. An error dialog pops up on the viewer and sometimes the client

**Expected behavior**

**Actual behavior**

**Screenshots**

Reported in

  https://groups.google.com/g/bigbluebutton-setup/c/U2XZFmyGBAI/m/rBZnSEZpBwAJ

See video from original post

https://user-images.githubusercontent.com/290351/211545505-9527e7a2-beb4-4a25-ba40-0a9c1e7e400b.mp4

A snapshot of the error dialog when testing with the latest build

<img width="380" alt="image" src="https://user-images.githubusercontent.com/290351/211545820-acc3bb02-4bac-4dbe-a75c-c14b351089fd.png">

**BBB version:**

```
BigBlueButton Server 2.6.0-beta.5 (679)
                    Kernel version: 5.4.0-122-generic
                      Distribution: Ubuntu 20.04.5 LTS (64-bit)
                            Memory: 8148 MB
                         CPU cores: 4
```

**Comment by ffdixon** [comment]
Looks like there is an incoming fix!  https://github.com/bigbluebutton/bigbluebutton/pull/16423

**Comment by ramonlsouza** [comment]
fixed by #16543
